config/sync/block.block.footer.yml

```diff
@@ -11,7 +11,7 @@ dependencies:
 id: footer
 theme: nicsdru_dept_theme
 region: footer
-weight: -25
+weight: -23
```

Comment:
I don't think this should be changed

Reply:
Is this to pop the footer links (copyright etc) above the department links?

Reply:
Oopsy fixed ☝🏻